node/string_decoder.ts

```diff
@@ -227,6 +228,16 @@ function simpleWrite(
   return buf.toString(this.encoding);
 }
 
+function simpleUtf8Write(
```

Comment:
Does it make sense to replace `utf8Write` with this implementation?

Reply:
no, that would require a stateful streaming decoder op. We can do that if we ever move `std/node` to the runtime :)

Reply:
ok, this only supports simple case (doesn't handle incomplete char bytes)

Reply:
> no, that would require a stateful streaming decoder op. We can do that if we ever move std/node to the runtime :)

Maybe we only need a little modified version of `core.decode`, which accepts 2 Uint8Arrays and returns a tuple of (string, remaining bytes)?